pages-e2e/fixtures/pages12.3.0/main.fixture

```diff
@@ -3,11 +3,12 @@
 		"simplePagesApiRoutes",
 		"simplePagesSsrRoutes",
 		"simplePagesStaticRoutesAndAssets",
-		"pagesMiddleware"
+		"pagesMiddleware",
+		"pagesRouting"
```

Comment:
I did not add the new `configRewritesRedirectsHeaders` to this fixture as it seems like such things don't work in Next 12.3.0 (not even with `next dev`)

I think it's fine not to include them in the tests, if later we end up having issues we might add 12.3.0 specific tests for whatever subset of such configs it supports

@james-elicx does Next 12.3.0 not supporting redirects and rewrites sounds correct to you? (especially with the `beforeFiles` and `afterFiles` options)

Reply:
The docs indicate that it should work, but if it's not working in `next dev`, well, there's not much we can do about it...

Reply:
well the docs don't specifically say when the `beforeFiles`, etc... were introduced 😕 🤷 

anyways even if they were I did saw other stuff that particularly in 12.3.0 is supposed to work but doesn't (in `next dev`)

so yeah I guess it's ok to leave this one be for now 🤷

Reply:
> there's not much we can do about it...

as I said we could check subsets of the configs that work... or try bumping the version slightly to newer patch versions to see if it helps... but doesn't really seem worth the effort here 🤔